cron/tasks/lmms/02-contributors

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 set -e
 
-git shortlog -sne | cut -c 8- > doc/CONTRIBUTORS
+git shortlog -sne > tmp/shortlog
```

Comment:
I tested `-o pipefail` and it does exactly what you're achieving without the temporary file.  Merged in https://github.com/LMMS/lmms.io/commit/e326760b990b539a46642b098ccff24fa774d6e7.